ocs_ci/utility/utils.py

```diff
@@ -3962,3 +3962,40 @@ def wipe_all_disk_partitions_for_node(node):
             if disk_to_wipe != root_disk:
                 disk_path = f"/dev/{disk_to_wipe}"
                 wipe_partition(node, disk_path)
+
+
+def get_ksctl_cli(bind_dir=None):
```

Comment:
There's no specific reason as such. The function for getting vault cli is also in `utils.py`. I can move it to the KMIP class if required

Reply:
`utils.py` was just a generic utility module that has become quite bloated. If we have a class/module that makes more sense like in this case I think it's better to move it there instead of leaving it in `utils.py`.